packages/core/strapi/lib/utils/ee.js

```diff
@@ -18,7 +18,7 @@ const internals = {};
 const features = {
   bronze: [],
   silver: [],
-  gold: ['sso'],
+  gold: ['sso', 'review-workflows'],
```

Comment:
I believe you will have some refacto to do for that following the #15147 PR that has been merged (ping @fdel-car)

Reply:
Yes, but not much, don't worry. Once you get the latest changes from `main`, you'll notice that this file has moved. You will need to update the `DEFAULT_FEATURES` object similarly to what you have done.

Side note: If needed, you can enrich your key with options specific to your feature.